event/event_test.go

```diff
@@ -197,6 +222,47 @@ var fromRequestTests = []struct {
 			Extensions:         map[string]interface{}{"myExtension": "ding"},
 		},
 	},
+	{
+		name: "valid CloudEvent in binary mode with valid event time",
+		requestHeaders: http.Header{
+			"Content-Type":          []string{"text/plain"},
+			"Ce-Eventtype":          []string{"myevent"},
+			"Ce-Eventtypeversion":   []string{"0.1beta"},
+			"Ce-Cloudeventsversion": []string{"0.1"},
+			"Ce-Source":             []string{"https://example.com"},
+			"Ce-Eventid":            []string{"778d495b-a29e-48f9-a438-a26de1e33515"},
+			"Ce-Eventtime":          []string{"1985-04-12T23:20:50.00Z"},
+			"Ce-Schemaurl":          []string{"https://example.com"},
+			"Ce-X-MyExtension":      []string{"ding"},
+		},
+		requestBody: []byte("hey there"),
+		expectedEvent: &eventpkg.Event{
+			EventType:          eventpkg.TypeName("myevent"),
+			EventTime:          &testTime,
+			CloudEventsVersion: "0.1",
+			Source:             "https://example.com",
+			ContentType:        "text/plain",
+			SchemaURL:          "https://example.com",
+			Data:               []byte("hey there"),
+			Extensions:         map[string]interface{}{"myExtension": "ding"},
+		},
+	},
+	{
+		name: "valid CloudEvent in binary mode with invalid event time",
+		requestHeaders: http.Header{
+			"Content-Type":          []string{"text/plain"},
+			"Ce-Eventtype":          []string{"myevent"},
```

Comment:
Should these be `CE-<name>` like the `events/events.go` package? I guess it doesn't really matter but for consistency...

Reply:
This is how headers are stored internally in http.Header struct.